portfolio/src/main/webapp/script.js

```diff
@@ -38,6 +42,28 @@ function createMap() {
       });
 }
 
+/**
+ * Draws a chart and adds it to the page.
+ */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('date', 'Year');
+  data.addColumn('number', 'Interest');
+        data.addRow([new Date(2020, 1), 85]);
+        data.addRow([new Date(2020, 2), 85]);
+        data.addRow([new Date(2020, 3), 97]);
+        data.addRow([new Date(2020, 4), 100]);
```

Comment:
The indentation for these lines looks pretty off.

Reply:
Done!